fedn/network/clients/connect.py

```diff
@@ -38,7 +38,7 @@ class ConnectorClient:
     :param force_ssl: True if https is used, False if http
     :type force_ssl: bool
     :param verify: True if certificate is verified, False if not
-    :type verify: bool
+    :type verify: str
```

Comment:
why is this changed to "str"?

Reply:
When I tried to install using pip3 install -e .(editable mode), it automatically switches to str, even after I changed to bool